### PR TITLE
skip applications install when generating workspaces.

### PR DESCRIPTION
**Pull request by mshima**
Current jdl generation makes npm workspaces unsuitable for auto install.
So disable it by default.

npm 8.<something> changed the workspaces behavior by deduping dependencies to the root.
We generate applications in parallel, so `install` will be executed 3 times in same npm tree causing lots of problems.
We need to execute `npm install` only once.

---

Please make sure the below checklist is followed for Pull Requests.

- [x] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
- [x] Tests are added where necessary
- [x] The JDL part is updated if necessary
- [x] [jhipster-online](https://github.com/jhipster/jhipster-online) is updated if necessary
- [x] Documentation is added/updated where necessary
- [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (below reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

File: cli/import-jdl.js [hunk 1]
```diff
@@ -534,6 +538,9 @@ module.exports = (jdlFiles, options = {}, env) => {
       .then(() => jdlImporter.generateEntities(env))
       .then(() => jdlImporter.generateDeployments())
       .then(() => {
+        if (options.workspaces) {
+          logger.log(`${chalk.green.bold("'npm install'")} was skipped due to workspaces. Run it by yourself.`);
```
Comment (mraible) [inline cli/import-jdl.js:542]:
```suggestion
          logger.log(`${chalk.green.bold("'npm install'")} was skipped due to workspaces. Run it by yourself.`);
```

"Run it by yourself" is a little brash. Can we make this more friendly?

**Comment by mraible** [comment]
I was able to fix my Docker issues using [this answer on Stack Overflow](https://stackoverflow.com/a/64620609/65681). 

After fixing Docker, I built each image and ran `npm run ci:e2e:prepare` at the top level. After everything was started, I tried:

```
npm run e2e --workspaces
```

This worked!

Screen Shot 2022-06-09 at 10 20 08

**Review by mraible — DISMISSED** [review]

**Comment by mshima** [comment]
@mraible now it skips install at applications, installs at the root.

Tested using:
```
jhipster jdl reactive-mf --workspaces
npm run java:docker:arm64
npm run ci:e2e:prepare
```
Then access `localhost:8080`

**Review by mraible — APPROVED** [review]

**Comment by mraible** [comment]
I love how fast the M1 is for e2e tests!

```
$ npm run e2e --workspaces
...
Execution time: 42 s.
```

**Pull request merged**